src/components/People.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from './PersonLink';
+
+export const People = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const getPeopleData = async () => {
+    setIsLoading(true);
+    try {
+      const peopleData = await getPeople();
+
+      setPeople(peopleData);
+    } catch {
+      setError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const getFaterSlug = (person: Person) => {
+    const searchedGuy = people.find(
+      somebody => somebody.name === person.fatherName,
+    );
+
+    return searchedGuy?.slug || '';
+  };
+
+  const getMotherSlug = (person: Person) => {
+    const searchedWoman = people.find(
+      somebody => somebody.name === person.motherName,
+    );
+
+    return searchedWoman?.slug || '';
+  };
```

Comment:
Would be better to prepare your people array inside `useEffect` and add `mother` and `father` to every person object